src/index.js

```diff
@@ -0,0 +1,945 @@
+    },
+    ru: {
+      shiftUp: 'Ctrl',
+      shiftDown: 'Ctrl',
+    },
+  },
+  {
+    class: ['MetaLeft', 'special'],
+    eng: {
+      shiftUp: 'Win',
+      shiftDown: 'Win',
+    },
+    ru: {
+      shiftUp: 'Win',
+      shiftDown: 'Win',
+    },
+  },
+  {
+    class: ['AltLeft', 'special'],
+    eng: {
+      shiftUp: 'Alt',
+      shiftDown: 'Alt',
+    },
+    ru: {
+      shiftUp: 'Alt',
+      shiftDown: 'Alt',
+    },
+  },
+  {
+    class: ['Space', 'special'],
+    eng: {
+      shiftUp: ' ',
+      shiftDown: ' ',
+    },
+    ru: {
+      shiftUp: ' ',
+      shiftDown: ' ',
+    },
+  },
+  {
+    class: ['AltRight', 'special'],
+    eng: {
+      shiftUp: 'Alt',
+      shiftDown: 'Alt',
+    },
+    ru: {
+      shiftUp: 'Alt',
+      shiftDown: 'Alt',
+    },
+  },
+  {
+    class: ['ArrowLeft', 'special'],
+    eng: {
+      shiftUp: '◄',
+      shiftDown: '◄',
+    },
+    ru: {
+      shiftUp: '◄',
+      shiftDown: '◄',
+    },
+  },
+  {
+    class: ['ArrowDown', 'special'],
+    eng: {
+      shiftUp: '▼',
+      shiftDown: '▼',
+    },
+    ru: {
+      shiftUp: '▼',
+      shiftDown: '▼',
+    },
+  },
+  {
+    class: ['ArrowRight', 'special'],
+    eng: {
+      shiftUp: '►',
+      shiftDown: '►',
+    },
+    ru: {
+      shiftUp: '►',
+      shiftDown: '►',
+    },
+  },
+  {
+    class: ['ControlRight', 'special'],
+    eng: {
+      shiftUp: 'Ctrl',
+      shiftDown: 'Ctrl',
+    },
+    ru: {
+      shiftUp: 'Ctrl',
+      shiftDown: 'Ctrl',
+    },
+  },
+];
+
+/* global document, localStorage */
+
+const rows = [keyRow1, keyRow2, keyRow3, keyRow4, keyRow5];
+
+const myBody = document.querySelector('body');
+
+// create wrapper for keyboard
+myBody.innerHTML =
+  '<section class="wrapper"><textarea id="text" placeholder="Write smth!" rows="15"></textarea><section class="keyboard"></section></section>';
+
+const createRow = keyRow => {
+  const row = document.createElement('div');
+  row.classList.add('row');
+
+  for (let i = 0; i < keyRow.length; i += 1) {
+    const myKey = document.createElement('div');
+    myKey.className = 'key';
+    myKey.classList.add(...keyRow[i].class);
+
+    myKey.innerHTML = `<span class="ru hide"><span class="shift up hide">${keyRow[i].ru.shiftUp}</span><span class="shift down hide">${keyRow[i].ru.shiftDown}</span></span><span class="eng"><span class="shift up">${keyRow[i].eng.shiftUp}</span><span class="shift down hide">${keyRow[i].eng.shiftDown}</span></span>`;
+    row.appendChild(myKey);
+  }
+  return row;
+};
+
+// creating keyboard
+const createKeyboard = rowsArr => {
+  for (let i = 0; i < rowsArr.length; i += 1) {
+    document.querySelector('.keyboard').appendChild(createRow(rowsArr[i]));
+  }
+};
+
+createKeyboard(rows);
+
+// ------------ KEYBOARD EVENTS ------------- //
+const keyboard = document.querySelector('.keyboard');
+const text = document.querySelector('#text');
+let shiftState = 'up';
+let lang = 'eng';
+
+function backspace(str, start, end) {
```

Comment:
`backspace` name isn't good name for function. naming convention please.

Reply:
What is `start` and `end` ? names of variable doesn't cover purpose what they suppose to do, it's better to rename them or add `js-doc` here.

Reply:
fixed